x-pack/solutions/observability/packages/kbn-observability-ui-semantic-conventions/jest.config.js

```diff
@@ -5,9 +5,10 @@
  * 2.0.
  */
 
-export enum ProcessorEvent {
-  transaction = 'transaction',
-  error = 'error',
-  metric = 'metric',
-  span = 'span',
-}
+module.exports = {
```

Comment:
GitHub is confused. processor_event.ts was deleted and semconv/jest.config.js was created.